### PR TITLE
Fixed label positions

### PR DESCRIPTION
**Pull request by ACovaci99**

### CODE REVIEW

**Comment by ACovaci99** [comment]
Ready for review

File: development/launch_gui.ipynb
Comment (vergauwenthomas) [inline development/launch_gui.ipynb:1]: @ACovaci99 remove this file from the commit.

File: examples/plotting_example.py [hunk 1]
```diff
@@ -17,8 +17,8 @@
 )
 
 
-# import sys
-# sys.path.append(str(main_folder))
+import sys
+sys.path.append(str(main_folder))
```
Comment (vergauwenthomas) [inline examples/plotting_example.py:21]: comment these lines (these should only be uncommented by developers, and not in the repo)

File: examples/plotting_example.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
 aug_2020_all_vlinders = metobs_toolkit.Dataset()
 aug_2020_all_vlinders.update_settings(
     input_data_file=testdata_file,  # A demo data file, downloaded with brian tool: https://vlinder.ugent.be/vlinderdata/multiple_vlinders.php
-    output_folder="/home/$USER/output/",
+    output_folder="C:/Users/andre/Desktop/test/",
```
Comment (vergauwenthomas) [inline examples/plotting_example.py:37]: `output_folder="/home/$USER/output/"`

File: examples/plotting_example.py [hunk 3]
```diff
@@ -81,14 +81,21 @@
 # To plot timeseries for one station you can use the make_plot function on the station object:
 favorite_station = aug_2020_all_vlinders.get_station(stationname="vlinder02")
 favorite_station.apply_quality_control()
+analysis = aug_2020_all_vlinders.get_analysis()
+analysis.get_diurnal_statistics(obstype='temp', stations=None,
+                           startdt=None, enddt=None, plot=True, colorby='name',
+                           errorbands=True, verbose=False)
+
+analysis.get_diurnal_statistics_with_reference(obstype='temp', refstation='vlinder01', tollerance='30T', stations=None,
+                           startdt=None, enddt=None, plot=True, colorby='name', errorbands=False, verbose=False)
```
Comment (vergauwenthomas) [inline examples/plotting_example.py:90]: Remove these lines. There will be a seperate example on diurnal analysis.

**Review by vergauwenthomas — CHANGES_REQUESTED** [review]

**Comment by vergauwenthomas** [comment]
@ACovaci99 I will fixe the review so i can deploy a new version

**Comment by vergauwenthomas** [comment]
resolved in #191

**Pull request closed without merging**